docs/src/modules/components/DemoErrorBoundary.js

```diff
@@ -22,7 +22,7 @@ export default class DemoErrorBoundary extends React.Component {
       return (
         <div>
           <Typography color="error" component="p" variant="h5" gutterBottom>
-            This demo had a runtime error!
+            {'This demo had a runtime error!'}
```

Comment:
Where is this change coming from?

Reply:
It was for `material-ui/no-hardcoded-labels`

Reply:
I will revert this change as no longer required.